### PR TITLE
Updated Advanced Redux

### PR DESCRIPTION
**Pull request by bttodd11**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -17,16 +19,18 @@ messages
 } = state;
 
 ReactDOM.render(
-  <App 
+  <Provider store = {store} >
+  <App
```
Comment (kdk745) [inline src/index.js]: Now that you're using redux, you won't have to pass the props into each component from App. Remember, we pass props directly into the components themselves using containers and mapStateToProps. Doing it again here is redundant.

File: src/store.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { createStore } from "redux";
+import state from "./state.js";
+import reducers from "./reducers";
+
+
+
+const store = createStore(state,reducers);
```
Comment (kdk745) [inline src/store.js]: Switch the state and reducers parameters inside your createStore()

File: src/components/TopNav.js [hunk 3]
```diff
@@ -66,4 +67,12 @@ function TopNav(props) {
     </ul>);
 }
 
-export default TopNav;
+
```
Comment (kdk745) [inline src/components/TopNav.js]: Good job here. This is where the props will be passed into the components now, so you can remove the props passed into them from the original parent component. It is now done in the TopNavContainer.

File: src/components/Comments.js [hunk 4]
```diff
@@ -26,4 +27,9 @@ function Comments(props) {
     </div>);
 }
 
-export default Comments;
+const mapStateToProps = function (state) {
+  return {
+    Comments: state.Comments
```
Comment (jw56578) [inline src/components/Comments.js]: Think more about this part.  The left side of the colon is suppose to be the name of the prop this component is using. Is it using a prop called Comments?

File: src/components/Orders.js [hunk 5]
```diff
@@ -26,4 +27,9 @@ function Orders(props) {
     </div>);
 }
 
-export default Orders;
+const mapStateToProps = function (state) {
+  return {
+    Orders: state.Orders
```
Comment (jw56578) [inline src/components/Orders.js]: Go back and review the videos and lesson and address book to remember the pattern.

**Comment by jw56578** [comment]
When you run the app, none of the numbers for comments, tasks, orders or tickets are showing up.
Can you figure out why?

**Pull request closed without merging**